viz/cmd/testdata/install_default.golden

```diff
@@ -391,52 +388,7 @@ metadata:
     linkerd.io/extension: viz
     component: web
     namespace: linkerd-viz
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
-metadata:
-  name: psp
-  namespace: linkerd-viz
-  labels:
-    linkerd.io/extension: viz
-rules:
-- apiGroups: ['policy', 'extensions']
-  resources: ['podsecuritypolicies']
-  verbs: ['use']
-  resourceNames:
-  - linkerd-linkerd-control-plane
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-  name: viz-psp
-  namespace: linkerd-viz
-  labels:
-    linkerd.io/extension: viz
-    namespace: linkerd-viz
-roleRef:
-  kind: Role
-  name: psp
-  apiGroup: rbac.authorization.k8s.io
-subjects:
-- kind: ServiceAccount
-  name: tap
-  namespace: linkerd-viz
-- kind: ServiceAccount
-  name: web
-  namespace: linkerd-viz
-- kind: ServiceAccount
-  name: grafana
-  namespace: linkerd-viz
-- kind: ServiceAccount
-  name: prometheus
-  namespace: linkerd-viz
-- kind: ServiceAccount
-  name: metrics-api
-  namespace: linkerd-viz
-- kind: ServiceAccount
-  name: tap-injector
-  namespace: linkerd-viz
+
```

Comment:
Tiny nit: It looks like the templating adds a new line here. This doesn't seem to affect this being applied to a cluster but I thought I should call it out.

Reply:
Nice catch :+1: